app/src/App/tests/App.test.tsx

```diff
@@ -179,4 +180,20 @@ describe('App - authenticated', () => {
     });
     wrapper.unmount();
   });
+
+  it('correctly logs out user', async () => {
+    const mock = jest.spyOn(serverLogout, 'logout');
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: `/logout` }]}>
+          <App />
+        </MemoryRouter>
+      </Provider>
+    );
+    await act(async () => {
+      await new Promise<unknown>((resolve) => setImmediate(resolve));
+      wrapper.update();
+    });
+    expect(mock).toHaveBeenCalled();
```

Comment:
Nice...do we need to check some text on the page to make sure we are on the right page?

Reply:
Not really, the logout component redirects to keycloak login page.